Lib/idlelib/configdialog.py

```diff
@@ -674,46 +675,27 @@ def var_changed_are_keys_builtin(self, *params):
     def var_changed_win_width(self, *params):
         "Store change to window width."
         value = self.win_width.get()
-        self.add_changed_item('main', 'EditorWindow', 'width', value)
+        changes.add_option('main', 'EditorWindow', 'width', value)
 
     def var_changed_win_height(self, *params):
         "Store change to window height."
         value = self.win_height.get()
-        self.add_changed_item('main', 'EditorWindow', 'height', value)
+        changes.add_option('main', 'EditorWindow', 'height', value)
 
     def var_changed_startup_edit(self, *params):
         "Store change to toggle for starting IDLE in the editor or shell."
         value = self.startup_edit.get()
-        self.add_changed_item('main', 'General', 'editor-on-startup', value)
+        changes.add_option('main', 'General', 'editor-on-startup', value)
 
     def var_changed_autosave(self, *params):
         "Store change to autosave."
         value = self.autosave.get()
-        self.add_changed_item('main', 'General', 'autosave', value)
+        changes.add_option('main', 'General', 'autosave', value)
 
     def var_changed_encoding(self, *params):
         "Store change to encoding."
         value = self.encoding.get()
-        self.add_changed_item('main', 'EditorWindow', 'encoding', value)
-
-    def reset_changed_items(self):
-        """Reset dictionary containing the items changed on each tab.
-
-        When any config item is changed in this dialog, an entry
-        should be made in the relevant section (config type) of this
-        dictionary. The key should be the config file section name and the
-        value a dictionary, whose key:value pairs are item=value pairs for
-        that config file section.
-        """
-        self.changed_items = {'main':{}, 'highlight':{}, 'keys':{},
-                             'extensions':{}}
-
-    def add_changed_item(self, typ, section, item, value):
-        "Add item/value pair to changed items dictionary for typ and section."
-        value = str(value)  # Make sure we use a string.
-        if section not in self.changed_items[typ]:
-            self.changed_items[typ][section] = {}
-        self.changed_items[typ][section][item] = value
+        changes.add_option('main', 'EditorWindow', 'encoding', value)
 
     def GetDefaultItems(self):
```

Comment:
I grepped the file for 'default' to make sure you had not changed the name elsewhere to somethingdefaultsomething.  55 hit, not orphan function call.  My guess is that someone wrote this, possibly for the check in save_option, and someone (same or other) decided to stick with idleConf instead of the skeletal duplicate, but did not remove.

I removed it in a separate commit.